news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "PrestaShop 1.7 minor release lifecycle"
+subtitle: "The journey of a minor release of PS 1.7, step by step"
+date:   2020-03-12 16:00:00
+authors: [ MathieuFerment ]
+icon: icon-code
+tags:
+ - releases
+ - 1.7
+ - beta
+---
+
+
+The journey of the release of PrestaShop 1.7.7.0 release has started. This moment is a nice opportunity to explain fully the PrestaShop minor releases lifecycle.
+
+# The journey begins: Kanban and scope
+
+Each minor version is defined by a feature scope, and this scope is defined as a number of github issues that we put into a [Kanban](https://help.github.com/en/github/managing-your-work-on-github/about-project-boards) (see [1.7.7 Kanban](https://github.com/PrestaShop/PrestaShop/projects/7)). How these issues are picked or sorted is the responsibility of the Product Team which spends a lot of time gathering feedback about all PrestaShop users to make sure the next minor version addresses the most important matters and needs.
+
+For example 1.7.7.0 scope contained, but not only:
+- Compatibility with php 7.3
+- Migration and rework of Back-office Order pages
+- Advanced currencies management
+
+Although it is very hard to estimate the size of this scope, we try to size it in order for the development phase to last 4 months.
+
+When this scope is completed - i.e all github issues have reached the "Done" column of the Kanban - we trigger the [Feature Freeze](https://en.wikipedia.org/wiki/Freeze_(software_engineering)).
+
+We however accept that some issues remain in the Kanban, outside of "Done" column, only if they are bugs, no features. This is a Feature Freeze, not a Bug Freeze.
+
+# Feature Freeze
+
+Feature freeze means that we freeze the development of the version and enter a phase of stabilization, whose aim is naturally the release some time later.
```

Comment:
I think the use of whose is wrong as it should refer to a person. "which" seems better but I let Pablo tell.